Automattic/wp-calypso
# Add JPO to Start framework - take 2

### This is still a work-in-progress.

The purpose of this PR is to add the Jetpack Onboarding process to Calypso. Users from specific hosts (or possibly under other conditions) are given a special connect URL that includes `onboarding=1`. Once the user has connected their Jetpack site, they will be automatically redirected directly to the JPO flow. It's still unclear where the user ends up after the flow is complete (TBD).

### Testing Instructions:
Boot up your Calypso dev on this branch.

Apply [this branch](https://github.com/Automattic/jetpack/pull/7468) to the Jetpack plugin on a Jetpack site. Disconnect and then reconnect that site to Jetpack. Grab the connection URL (`/jetpack/connect/authorize/.....`) during the process and change `https://wordpress.com` to `http://calypso.localhost:3000` in the URL.

Paste the connection URL in your browser and follow the normal process to connect Jetpack. You should then be taken to the onboarding flow.
